src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -393,4 +411,13 @@ private void verifyEmail(EmailWrapper email, String recipient, String subject, S
         assertFalse(emailContent.contains("${"));
     }
 
+    private boolean verifyCoOwners(List<InstructorAttributes> coOwners) {
```

Comment:
Not sure why the pmd is acting up here. But the way you are testing the method can be improved. It is odd that this method is used specifically in testGenerateStudentCourseJoinEmail(). Perhaps you can just create a method called `testGenerateCoOwners...` (depending on what you name the method in the end) in the test. You could also include testing whether all co-owners of the particular course are included in the line.